Specs/Scene/ModelSpec.js

```diff
@@ -88,7 +88,9 @@ defineSuite([
     var boxSpotLightUrl = './Data/Models/MaterialsCommon/BoxSpotLight.gltf';
     var boxTransparentUrl = './Data/Models/MaterialsCommon/BoxTransparent.gltf';
     var boxColorUrl = './Data/Models/Box-Color/Box-Color.gltf';
+    var boxUint32Indices = './Data/Models/Box-Uint32Indices/Box-Uint32Indices.gltf';
     var boxQuantizedUrl = './Data/Models/WEB3DQuantizedAttributes/Box-Quantized.gltf';
+    var boxNormalizedQuantizedUrl = './Data/Models/WEB3DQuantizedAttributes/Box-Normalized-Quantized.gltf';
```

Comment:
Can we build new test models (in this case, it is good for this to be separate since it requires an extension) on a super simple model without quantization so test failures are easier to track down in the more limited scope?